app/src/main/java/com/infomaniak/drive/ui/fileList/FileAdapter.kt

```diff
@@ -443,6 +503,17 @@ open class FileAdapter(
         LIST(R.layout.cardview_file_list)
     }
 
+    data class FileItemViews(
```

Comment:
Would adding the keyword "common" to qualify the views make sens in the name of the class? Do you have an opinion on the subject? Like, to clearly state that these are the views that are common to all three layouts. All of these views are accessible at all time by simply accessing the binding but the ones inside the data class are there for when we want to do something no matter what layout we're dealing with

Reply:
Well this is technically not the case, fileDate and StopUploadButton aren't shared with all bindings. But removing it would make the need of different binding cast inside this adapter, which complexify the code

Reply:
However, we could extends the when that you're refering to in your previous comment to avoid other cast, to the cost of a bit more logic duplication. I though it was clearer like this, to keep the when to its simplest form.
Another solution would be to hard cast without verification for exemple fileDate, as it exists only for FileListItemBInding, but I find this less reliable and more error prone in the future

Reply:
We definitely want to keep the nullable views inside FileItemViews. But I don't think it goes against naming it "common" nonetheless